static/js/queues.js

```diff
@@ -125,10 +125,12 @@ document.querySelector('#declare').addEventListener('submit', function (evt) {
     arguments: Dom.parseJSON(data.get('arguments'))
   }
   HTTP.request('PUT', url, { body })
-    .then(() => {
+    .then((response) => {
       queuesTable.reload()
       evt.target.reset()
-      Dom.toast('Queue ' + queue + ' created')
+      if ((typeof response !== 'undefined')) {
```

Comment:
Would this work? 
```suggestion
      if (response) {
```

Reply:
Yes! 
But I'm thinking that this works because the endpoint currently dosn't return anything. We could add an is_error value to the error response just to be explicit with that it is an error (see my latest commit).